crates/macro/src/lib.rs

```diff
@@ -18,6 +18,26 @@ pub fn wasm_bindgen(attr: TokenStream, input: TokenStream) -> TokenStream {
     }
 }
 
+/// This macro adds a linked module by module, raw_module or inline_js attribute.
+/// It expands to a String containing a link to that module. This link may be relative and
```

Comment:
I think I will make the output URLs always absolute. Relative URLs seemed nice because they usually can be used as paths in node. On the other hand, ensuring correctness even with `chdir` or `history.replace` seems difficult. More importantly, in the case when you `chdir` to another drive on Windows, you cannot return a string that is both a path and a relative URL referring to a linked module. Therefore, `link_to!` should just return absolute URLs, and users on node can simply call `url.pathToFileURL` if needed.